### PR TITLE
Fider chart

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

File: fider/templates/secret-jwt-token.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: {{ template "fider.fullname" . }}-jwt-secret
+  labels:
+    app: {{ template "fider.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  jwt-secret: {{ randAlphaNum 64 | b64enc | quote  }}
```
Comment (wenzowski) [inline fider/templates/secret-jwt-token.yaml:12]: This is coming from [`crypto/rand`](https://golang.org/pkg/crypto/rand/) right? Helm is dependent on Sprig for this?

Comment (wenzowski) [reply fider/templates/secret-jwt-token.yaml:12]: I'm fussing here to make sure we've picked a CSPRNG and not just a PRNG

**Review by wenzowski — COMMENTED** [review]
This looks amazing!
I want to be absolutely sure about the one comment before firing off approval.

File: fider/templates/secret-jwt-token.yaml (hunk 1 above)
Comment (matthieu-foucault) [reply fider/templates/secret-jwt-token.yaml:12]: Yes helm uses sprig: https://github.com/helm/helm-www/blob/master/content/en/docs/chart_template_guide/functions_and_pipelines.md

http://masterminds.github.io/sprig/strings.html
> #### randAlphaNum, randAlpha, randNumeric, and randAscii
> These four functions generate cryptographically secure (uses crypto/rand) random strings

**Review by wenzowski — APPROVED** [review]
:shipit:

**Pull request merged**